examples/localhost.yaml

```diff
@@ -28,3 +34,40 @@ policies:
     name: deny-9999
     namespace: default
     scope: Namespace
+services:
+- name: local
+  namespace: default
+  hostname: example.com
+  addresses:
+    - /127.10.0.1
+  endpoints:
```

Comment:
seems like in our yaml we should either use `vips` or `endpoints`, needing to specify both seems tedious?

Reply:
not sure i understand the comment here, so ill elaborate on the intent for now:

so this is test config for our internal `Service` struct, which has both the service vips (in `addresses` and the endpoints behind the service `endpoints` which are basically the workloads)

this `Service` struct is constructed internally after we receive the xds workloads and services, in whatever order. if the xds service arrives we fill endpoints with all workloads it selects we currently know about

if the xds workload arrives first, we add its virtual IPs to `staged_vips` internal map so if/when the corresponding xds service arrives we can populate this internal `Service` struct with the right endpoints

Reply:
This doesn't _have_ to be 1:1 with the XDS struct. Even in the workload we already have LocalWorkload which differs

Reply:
ah I see what you mean, makes sense :+1: . I'll take a look at that update

Reply:
addressed in https://github.com/istio/ztunnel/pull/486/commits/b5c28c2bf3503dcdf79b1f6a1e23a695e45681e5